chain/network/src/network_protocol/borsh.rs

```diff
@@ -81,7 +81,7 @@ impl From<HandshakeAutoDes> for Handshake {
 }
 
 #[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
-#[derive(BorshSerialize, BorshDeserialize, PartialEq, Eq, Clone, Debug)]
+#[derive(Default, BorshSerialize, BorshDeserialize, PartialEq, Eq, Clone, Debug)]
```

Comment:
I am wondering, should we drop borsh already? I think we had a release with protobufs already, no? 

Unrelated to the PR

Reply:
yes, protobuf support is available since version 54, and 53 is already unsupported. I'll do that soon.